extensions/imports/config/FBRequireConfig.js

```diff
@@ -0,0 +1,35 @@
+const {compareStrings, isCapitalized} = require('nuclide-format-js-base/lib/utils/StringUtils');
+const getDeclarationName = require('../utils/getDeclarationName');
+const isGlobal = require('nuclide-format-js-base/lib/utils/isGlobal');
+const isValidRequireDeclaration = require('../utils/isValidRequireDeclaration');
+const jscs = require('jscodeshift');
+
+module.exports = [
+  // Handle UpperCase requires, e.g: `require('UpperCase');`
+  {
+    searchTerms: [jscs.VariableDeclaration],
+    filters: [
+      isGlobal,
+      path => isValidRequireDeclaration(path.node),
+      path => isCapitalized(getDeclarationName(path.node)),
+    ],
+    comparator: (node1, node2) => compareStrings(
+      getDeclarationName(node1),
+      getDeclarationName(node2)
+    ),
+  },
+
+  // Handle lowerCase requires, e.g: `require('lowerCase');`
+  {
+    searchTerms: [jscs.VariableDeclaration],
+    filters: [
+      isGlobal,
+      path => isValidRequireDeclaration(path.node),
+      path => !isCapitalized(getDeclarationName(path.node)),
+    ],
+    comparator: (node1, node2) => compareStrings(
+      getDeclarationName(node1),
+      getDeclarationName(node2)
+    ),
+  },
```

Comment:
@kyldvs would be great to expose these configs from `format-js-base` so i didnt need to recreate them here. https://github.com/facebook/nuclide/blob/master/pkg/nuclide/format-js-base/lib/requires/formatRequires.js#L32-L92